bridge/server/app.ts

```diff
@@ -192,7 +192,7 @@ async function init(): Promise<Express> {
     }
 
     res.status(err.response?.status || 500).send(err.message);
-    console.error(err);
+    console.error(`[ERROR] ${err.message}`);
```

Comment:
I would also log the requested URL and status code here if they exist. Without this, it is not possible to see where this error came from

Reply:
The whole stack trace is also lost. We should differentiate if the error is an instance of Error or not.

Reply:
I would also change the err type of `any` to `Error | AxiosError`

Reply:
Sure, we can differentiate `Error` which might happen in the application and `AxiosError` which come from API calls.